src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,15 +1,52 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
+# TO-DO: complete the helper function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
     # TO-DO
-    
+    # put back together here
+    # Sorting happens here
+
+    a = 0
+    b = 0 
+
+    for k in range(0, elements):
```

Comment:
Some of these comments don't line up with what your code is actually doing. I'd go through and fix this to avoid confusion for future you.